### ISSUE TITLE
config.ini values get ignored

### ISSUE THREAD

**Issue by JohnTheodore** [metagriffin/dnssync#3]
It doesn't seem like I can specify domain in config.ini, and do 'dnssync download --config config.ini', I have to explicitly do --domain

**Comment by metagriffin** [comment]
yeah, there are several weirdnesses involved here -- primarily due to the fact that the config file was actually an after-thought... specifically, in this case the problem is that the config file can support multiple domains.

i'll give this some thought, and figure out how to do this "correctly".

thanks for the feedback and please let me know if you run into any other issues.
